src/pybind/mgr/dashboard/grafana.py

```diff
@@ -18,10 +19,14 @@ class GrafanaRestClient(object):
 
     @staticmethod
     def url_validation(method, path):
-        response = requests.request(
-            method,
-            path,
-            verify=Settings.GRAFANA_API_SSL_VERIFY)
+        try:
+            response = requests.request(
+                method,
+                path,
+                verify=Settings.GRAFANA_API_SSL_VERIFY)
+        except SSLError:
+            logger.exception("Failed to verify Grafana dashboard for URL %s", path)
```

Comment:
Dito here - I know that we perform a "validation" here, by accessing the respective URL. How about using "load" here?
```suggestion
            logger.exception("Failed to load Grafana dashboard for URL %s", path)
```